### ISSUE TITLE
[Bug] It does not load me anything, when I open the application

### ISSUE THREAD

**Issue by Shirvo12** [YTVanced/Vanced#537]
![Screenshot_20210307-100656](https://user-images.githubusercontent.com/80235455/110272273-2c520600-7fa0-11eb-81cd-3baa6c9547f4.jpg)

**Bug description**
Replace me with a clear and detailed description of the bug.

**Variant**
Non-Root/Root

**Vanced version**
16.02.35 - Build - 01.47.00

**Android version**
Android 9.0 - Pie

**Device**
Samsung Galaxy S8

**Steps to Reproduce**
Steps to reproduce the error:
1. Open the app
3. ....

**Expected behavior**
Replace me with a description of what you expected should happen and what happened instead.

_Attach images/videos if possible. These can be helpful in solving the issue._

I've been installing and trying to fix the problem for hours and I can't :( help me

**Comment by sagarsoni7** [comment]
++

**Comment by smed79** [comment]
I Am facing to the same issue after deleting the app data.
```
Android 10
YouTube Vanced 16.02.35
Vanced Manager 2.5.1
Vanced microG 0.2.16.204713
```

**Comment by Shirvo12** [comment]
Lo resolví descarga una versión antigua antes de la reciente que salió  que es la 16.02.35 descarga al versión anterior a esa.
Traducelo al español.

**Comment by smed79** [comment]
Fixed here by downgrading to **v15.43.32** (_updating to **v16.02.35** break the app_).
